lib/active_interactor/interactor/callbacks.rb

```diff
@@ -249,6 +249,22 @@ def before_rollback(*filters, &block)
           set_callback(:rollback, :before, *filters, &block)
         end
 
+        # Set {.after_callbacks_deferred_when_organized} to `true`
+        #
+        # @example a basic {Base organizer} set to defer 'after' callbacks when organized
+        #   class MyOrganizer < ActiveInteractor::Organizer::Base
+        #     defer_after_callbacks_when_organized
+        #   end
+        def defer_after_callbacks_when_organized
```

Comment:
do we need this method on Interactors or is this something we can just have on an organizer?

Reply:
Yeah, good question. My thinking was to add it here so that each Interactor could independently elect to defer or not for maximum, per-Interactor flexibility. That being said, I'd be down to move it to the Organizer (as `defer_after_interactor_callbacks` or something like that) if you think that would be more straightforward. Let me know which you'd prefer

Reply:
Actually I tend to agree with you here. This is fine to leave on the interactor, but I do wonder if there is a use case for an interactor to behave one way when organized with organizer A and a different way when organized with organizer B.

Reply:
I suppose I could add a config in the Organizer in addition to the one in the Interactor so that it can be set in either place.

Reply:
Shouldn't be a need the organizer will already inherit this callback.

Reply:
Oh yeah, good point. So when it's set on the `Organizer` does it automatically get set on every `Interactor` as well? Or do I need to add some extra handling logic in the `Organizer` perform? I'll add a spec so I can test it out...

Reply:
Not quite [ActiveInteractor::Organizer::Base](https://github.com/aaronmallen/activeinteractor/blob/main/lib/active_interactor/organizer/base.rb#L9) is itself an interactor.  So it will inherit all of the call backs `ActiveInteractor::Base` would have meaning if an organizer was organized inside of another organizer this callback would apply.

Reply:
Oh, I gotchya. I wasn't thinking about organized Organizers, but that's a good call. I'll add a spec for that

Reply:
Okay, pushed up a spec for that.